docs/configuration.md

```diff
@@ -162,6 +163,8 @@ Currently Catch enables `CATCH_CONFIG_WINDOWS_SEH` only when compiled with MSVC,
 
 `CATCH_CONFIG_POSIX_SIGNALS` is on by default, except when Catch is compiled under `Cygwin`, where it is disabled by default (but can be force-enabled by defining `CATCH_CONFIG_POSIX_SIGNALS`).
 
+`CATCH_CONFIG_GETENV` is on by default, except when Catch is compiled under some platforms that lacks working `getenv` (currently Windows UWP and Playstation).
```

Comment:
```suggestion
`CATCH_CONFIG_GETENV` is on by default, except when Catch2 is compiled for
platforms that lacks working `std::getenv` (currently Windows UWP and
Playstation).
```

Catch2 as the project name is preferred in newly (re)written documentation, same for 80 char columns.